graphx/src/main/scala/org/apache/spark/graphx/lib/PageRank.scala

```diff
@@ -196,11 +196,45 @@ object PageRank extends Logging {
     require(sources.nonEmpty, s"The list of sources must be non-empty," +
       s" but got ${sources.mkString("[", ",", "]")}")
 
+    val sourcesWithScores = sources zip Array.fill(sources.size)(1.0)
```

Comment:
Let's use explicit '.' notation
But this can be `sources.map((_, 1.0))` instead

Reply:
Done